front/components/navigation/Navigation.tsx

```diff
@@ -107,15 +111,39 @@ export function Navigation({
         </Dialog>
       </Transition.Root>
 
-      {/* Desktop sidebar */}
-      <div className="hidden lg:visible lg:inset-y-0 lg:z-0 lg:flex lg:w-80 lg:flex-col">
-        <NavigationSidebar
-          owner={owner}
-          subscription={subscription}
-          subNavigation={subNavigation}
-        >
-          {navChildren && navChildren}
-        </NavigationSidebar>
+      {/*Desktop sidebar*/}
+      <Transition
+        show={navigationBar}
+        as={Fragment}
+        enter="transition-all duration-2000 ease-out"
```

Comment:
I'm not sure `duration-2000` is in our safelist. I guess it defaults to 150ms. Let's fix it, right?